src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,21 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    enum Color {
+        RED,
+        BLUE,
+        PURPLE,
+        YELLOW,
+        BLACK,
+        ORANGE,
+        WHITE
+    }
+
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Color.values().length);
+        Color color = Color.values()[index];
+        return color.toString();
```

Comment:
use `.name()` not `.toString()`, its mentioned in checklist

Reply:
not fixed: use .name() not .toString(), its mentioned in checklist